imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -114,22 +183,101 @@
    "metadata": {
     "colab": {},
     "colab_type": "code",
-    "id": "UW3Y6Pd6d1Ey"
+    "id": "biHU0WoLjgYs"
    },
    "outputs": [],
    "source": [
-    "%%bash -s {project_id} {location} {automl_bucket_name}\n",
-    "# Create bucket.\n",
-    "gsutil -q mb -c regional -l $2 $3\n",
+    "import time\n",
     "\n",
-    "# Allow Cloud Healthcare API to write to bucket.\n",
-    "PROJECT_NUMBER=`gcloud projects describe chc-tcia | grep projectNumber | sed 's/[^0-9]//g'`\n",
-    "SERVICE_ACCOUNT=\"service-${PROJECT_NUMBER}@gcp-sa-healthcare.iam.gserviceaccount.com\"\n",
```

Comment:
This is still needed. But instead of CHC-TCIA its $1, because we are doing an export still.

Reply:
you mean https://github.com/GoogleCloudPlatform/healthcare/pull/443/files/eca5127e40dcf774337e224b993dcba1dec5ec0e#diff-f6a4270c35f32aba098eef15efc8f250R103 ?
service account have pubsub.admin(includes pubsub.publisher https://cloud.google.com/pubsub/docs/access-control) permissions and objectCreator

Reply:
service-${PROJECT_NUMBER}@gcp-sa-healthcare.iam.gserviceaccount.com(this is the one the CHC API uses when it exports vs ${PROJECT_NUMBER}-compute@developer.gserviceaccount.com,)

Reply:
done